assets/css/planning.scss

```diff
@@ -169,3 +169,15 @@ $tableBoxSize: 40px;
     }
   }
 }
+
+@media screen and (min-width: 1001px) {
+  .planning-actions-container {
+    position: sticky;
+    top: 0;
```

Comment:
Why are you hiding all the table sticky headers ?

Reply:
thoses rules works with the rules in `assets/css/availability-table.scss`, to have both form actions and table headers sticky